app/services/normalise-charge.js

```diff
@@ -69,13 +69,7 @@ module.exports = (function () {
       country: body.addressCountry
     }
   }
-  // body is passed by reference
-  const addressLines = function (body) {
-    if (!body.addressLine1 && body.addressLine2) {
-      body.addressLine1 = body.addressLine2
-      delete body.addressLine2
-    }
-  }
```

Comment:
I assume this is:

> Remove code that would combine address line 1 & 2 together. This code is no longer
required because each line now has its own field label.

However, it looks like that’s not quite what it was doing. What would happen was if there was no text for address line 1 but there was for address line 2, it would shift the address line 2 text up to address line 1. We’re losing that behaviour now. That might be fine but I want to check we understand what we’re doing and why.

Reply:
Good point - we removed this code as I didn't want `address line 2` shifting to `address line 1` anymore.
As we now have validation for `address line 1` and `address line 2` individually.